src/applications/disability-benefits/686c-674-v2/config/utilities.js

```diff
@@ -117,3 +118,24 @@ export function customTransformForSubmit(formConfig, form) {
 
   return JSON.stringify(withoutInactivePages, customFormReplacer) || '{}';
 }
+
+/**
+ * parseDateToDateObj from ISO8601 or JS number date (not unix time)
+ * @param {string, number, Date} date - date to format
+ * @returns {dateObj|null} date object
+ */
+export const parseDateToDateObj = (date, template) => {
```

Comment:
this feels overly complicated. Do we not know what format the Veteran's birthdate is in?

Reply:
I think he copied this function from me. Date-fns needs to parse ISO dates without the timestamp, otherwise it results in a date set to the current timezone. We were seeing CI errors because it runs in a different timezone.